odra-casper/test-vm/src/vm/casper_vm.rs

```diff
@@ -272,8 +273,8 @@ impl CasperVm {
     }
 
     /// Returns the report of the gas used during the whole lifetime of the CasperVM.
-    pub fn gas_report(&self) -> Vec<(String, U512)> {
-        self.gas_cost.clone()
+    pub fn gas_report(&self) -> GasReport {
```

Comment:
It is read-only I guess. You could return a ref instead of copy.

Reply:
In host I still need to clone it, so does it make a difference?
Any idea to improve this?
```
error[E0515]: cannot return value referencing temporary value
   --> /home/kuba/Projekty/odra/odra/core/src/host.rs:315:9
    |
315 |         self.backend.borrow().gas_report()
    |         ---------------------^^^^^^^^^^^^^
    |         |
    |         returns a value referencing data owned by the current function
    |         temporary value created here

```